lib/frozen_record/test_helper.rb

```diff
@@ -0,0 +1,41 @@
+module FrozenRecord
+  module TestHelper
+    NoFixturesLoaded = Class.new(StandardError)
+
+    def self.load_fixture(model_class, alternate_base_path)
+      @cache ||= {}
+
+      ensure_model_class_is_frozenrecord(model_class)
+
+      @cache[model_class] = {
+        old_base_path: model_class.base_path,
+        old_auto_reloading: model_class.auto_reloading,
+      }
+
+      model_class.auto_reloading = true
+      model_class.base_path = alternate_base_path
+    end
+
+    def self.unload_fixtures
+      if !defined?(@cache) || !@cache.present?
+        raise NoFixturesLoaded, "`unload_fixtures` was called before any calls to `load_fixture`"
```

Comment:
I think a case could be made to not raise if no fixtures were loaded.

My biggest worry with that feature is that people might misuse it and leak fixtures from one test to the other.

If you can call `unload_fixtures` regardless of wether fixtures were added or not, it allows you to call it in all your tests as part of `teardown`, which is a good way to ensure people can't mess it up.

Reply:
💯 Agreed. Running `unload_fixtures` as part of `teardown` for test cases that didn't load fixtures shouldn't force devs to rescue something like `NoFixturesLoaded`.